cmd/check-gh-automation/main.go

```diff
@@ -189,9 +189,16 @@ func checkRepos(repos []string, bots []string, appName string, ignore sets.Set[s
 		}
 
 		orgConfig := prowAgent.Config().BranchProtection.GetOrg(org)
-		var branchProtectionEnabled bool
+		branchProtectionEnabled := true // By default, it is turned on in absence of configuration stating otherwise
```

Comment:
Changing to default to `true`, I'm wondering if we are going to hit any error. Has this tool a `dry-run` mode? Alternatively we can merge and see what happens.

Reply:
It's always dry as it doesn't mutate anything. I tested it out with a few repos to make sure the behavior was correct. The default should have always been true, as that matches with the documentation on branch protection. This feature was never working as intended.